dev-docs/bidders/ventes.md

```diff
@@ -2,7 +2,7 @@
 layout: bidder
 title: ventes
 description: Prebid ventes Bidder Adapter
-pbjs: true
+pbjs: false
```

Comment:
@jessoventes did you mean to set this to false. There is a pbjs bid adapter

Reply:
Yes, we created a ventesBidAdapter.js -> https://github.com/prebid/Prebid.js/blob/master/modules/ventesBidAdapter.js. Need to set pbjs: true, do i need to make a new commit.